app/imports/api/methods/commentsMethods.js

```diff
@@ -0,0 +1,53 @@
+import { Meteor } from 'meteor/meteor'
+
+import { Applications } from '/imports/collections/applications.js';
+import { Comments } from '/imports/collections/comments.js';
+
+import { AccountsCommon } from 'meteor/accounts-base';
+
+export const submitComment = new ValidatedMethod({
+  name: 'comment.submit',
+
+  validate: new SimpleSchema({
+    applicationId: {
+      type: String,
+    },
+    comment: {
+      type: String,
+    },
+  }).validator(),
+
+  run({ applicationId, comment }) {
+    if (!Roles.userIsInRole(this.userId, ['committee', 'admin'])) {
+      throw new Meteor.Error('Coments.methods.submit.not-allowed', 'Must be committee to do this.');
+    }
+
+    let user = Meteor.users.findOne(this.userId);
+
+    Comments.insert({
+      userId: this.userId,
+      email: user.emails[0].address,
```

Comment:
i think we should add the `Name` field and add that to this part in a diff branch